### PR TITLE
#2255 Update version of master branch to 0.8.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2255 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
@jackyoh please unlock the QA for 0.8

**Comment by jackyoh** [comment]
OK, 我修改好了

**Comment by jackyoh** [comment]
retry

File: docs/src/contrib-docs/build_docs.rst [hunk 1]
```diff
@@ -98,7 +98,7 @@ Build docs
     [docs]$ make html
     Running Sphinx v1.8.5
     ========================================
-    Ohara version: 0.7.0-SNAPSHOT
+    Ohara version: 0.8.0-SNAPSHOT
```
Comment (vitojeng) [inline docs/src/contrib-docs/build_docs.rst]: Just a message, no need to change.

File: docs/src/rest/rest-stream.rst [hunk 2]
```diff
@@ -158,7 +158,7 @@ All default value response
         "from": [],
         "to": [],
         "instances": 1,
-        "imageName": "oharastream/streamapp:0.7.0-SNAPSHOT",
+        "imageName": "oharastream/streamapp:0.8.0-SNAPSHOT",
```
Comment (vitojeng) [inline docs/src/rest/rest-stream.rst]: Just a message, no need to change.

File: docs/src/user_guide.rst [hunk 3]
```diff
@@ -64,7 +64,7 @@ Configurator via following docker command.
 
 .. code:: bash
 
-   docker run --rm -p 12345:12345 oharastream/configurator:0.7.0-SNAPSHOT --port 12345
+   docker run --rm -p 12345:12345 oharastream/configurator:0.8.0-SNAPSHOT --port 12345
```
Comment (vitojeng) [inline docs/src/user_guide.rst]: Will file another PR to fix with `$|version|`

File: docs/src/user_guide.rst [hunk 4]
```diff
@@ -75,7 +75,7 @@ on above Ohara Configurator.
 
 .. code:: bash
 
-   docker run --rm -p 5050:5050 oharastream/manager:0.7.0-SNAPSHOT --port 5050 --configurator http://$ip:12345/v0
+   docker run --rm -p 5050:5050 oharastream/manager:0.8.0-SNAPSHOT --port 5050 --configurator http://$ip:12345/v0
```
Comment (vitojeng) [inline docs/src/user_guide.rst]: Will file another PR to fix with `$|version|`

File: docs/src/user_guide.rst [hunk 5]
```diff
@@ -165,7 +165,7 @@ Execute Configurator
 
 .. code:: sh
 
-   docker run --rm -p ${port}:${port} --add-host ${nodeHostName}:${nodeHostIP} oharastream/configurator:0.7.0-SNAPSHOT --port ${port} --hostname ${host}
+   docker run --rm -p ${port}:${port} --add-host ${nodeHostName}:${nodeHostIP} oharastream/configurator:0.8.0-SNAPSHOT --port ${port} --hostname ${host}
```
Comment (vitojeng) [inline docs/src/user_guide.rst]: Will file another PR to fix with `$|version|`

File: docs/src/user_guide.rst [hunk 6]
```diff
@@ -208,7 +208,7 @@ is to mount a local folder (/tmp/configurator) on
    $ mkdir /tmp/configurator
    $ docker run -v /tmp/configurator:/home/ohara/configurator \
             -p 12345:12345 \
-            oharastream/configurator:0.7.0-SNAPSHOT \
+            oharastream/configurator:0.8.0-SNAPSHOT \
```
Comment (vitojeng) [inline docs/src/user_guide.rst]: Will file another PR to fix with `$|version|`

File: docs/src/user_guide.rst [hunk 7]
```diff
@@ -224,7 +224,7 @@ Execute Manager
 
 .. code:: sh
 
-   docker run --rm -p 5050:5050 oharastream/manager:0.7.0-SNAPSHOT --port 5050 --configurator http://localhost:12345/v0
+   docker run --rm -p 5050:5050 oharastream/manager:0.8.0-SNAPSHOT --port 5050 --configurator http://localhost:12345/v0
```
Comment (vitojeng) [inline docs/src/user_guide.rst]: Will file another PR to fix with `$|version|`

File: docs/src/user_guide.rst [hunk 8]
```diff
@@ -251,7 +251,7 @@ Execute FTP Instance
 
 .. code:: bash
 
-   docker run --rm -p 10000-10011:10000-10011 oharastream/backend:0.7.0-SNAPSHOT com.island.ohara.testing.service.FtpServer --controlPort 10000 --dataPorts 10001-10011 --user ${UserName} --password ${Password} --hostname ${hostIP or hostName}
+   docker run --rm -p 10000-10011:10000-10011 oharastream/backend:0.8.0-SNAPSHOT com.island.ohara.testing.service.FtpServer --controlPort 10000 --dataPorts 10001-10011 --user ${UserName} --password ${Password} --hostname ${hostIP or hostName}
```
Comment (vitojeng) [inline docs/src/user_guide.rst]: Will file another PR to fix with `$|version|`

**Comment by chia7712** [comment]
+1

the 0.8.0-SNAPSHOT is nonexistent so the api test fails. @wu87988622 feel free to merge this one :)

File: docs/src/rest/rest-stream.rst [hunk 9]
```diff
@@ -325,7 +325,7 @@ And the response:
         "group": "default",
         "tags": {},
         "instances": 1,
-        "imageName": "oharastream/streamapp:0.7.0-SNAPSHOT",
+        "imageName": "oharastream/streamapp:0.8.0-SNAPSHOT",
```
Comment (vitojeng) [inline docs/src/rest/rest-stream.rst]: Just a message, no need to change.

**Comment by wu87988622** [comment]
> +1
> 
> the 0.8.0-SNAPSHOT is nonexistent so the api test fails. @wu87988622 feel free to merge this one :)

ok~ thanks.

File: docs/src/contrib-docs/build_docs.rst (hunk 1 above)
Comment (wu87988622) [reply docs/src/contrib-docs/build_docs.rst]: ok~ Thanks

File: docs/src/rest/rest-stream.rst (hunk 2 above)
Comment (wu87988622) [reply docs/src/rest/rest-stream.rst]: ok~ Thanks

File: docs/src/user_guide.rst (hunk 3 above)
Comment (wu87988622) [reply docs/src/user_guide.rst]: ok~ Thanks

File: docs/src/user_guide.rst (hunk 4 above)
Comment (wu87988622) [reply docs/src/user_guide.rst]: ok~ Thanks

File: docs/src/user_guide.rst (hunk 5 above)
Comment (wu87988622) [reply docs/src/user_guide.rst]: ok~ Thanks

File: docs/src/user_guide.rst (hunk 6 above)
Comment (wu87988622) [reply docs/src/user_guide.rst]: ok~ Thanks

File: docs/src/user_guide.rst (hunk 7 above)
Comment (wu87988622) [reply docs/src/user_guide.rst]: ok~ Thanks

File: docs/src/user_guide.rst (hunk 8 above)
Comment (wu87988622) [reply docs/src/user_guide.rst]: ok~ Thanks

File: docs/src/rest/rest-stream.rst (hunk 9 above)
Comment (wu87988622) [reply docs/src/rest/rest-stream.rst]: ok~ Thanks

**Pull request merged**